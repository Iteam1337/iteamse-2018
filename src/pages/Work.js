// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/WorkPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import { filterByLocation } from '../utils/filterByLocation'
import FilterByLocation from '../components/FilterByLocation/FilterByLocation'
import Header from '../components/Header/Header'
import GridRow from '../components/Grid/GridRow'
import Block from '../components/Blocks/Block'
import Team from '../components/Team/Team'

type Props = Iteam.ApolloBase<IteamCMS.WorkPage>

export const WorkPageQuery = gql`
  query WorkPage {
    pageWork {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
    }
    openpositions {
      id
      location
      role
      title
    }
  }
`

const Work = () => {
  return (
    <Query query={WorkPageQuery}>
      {({ loading, data: { openpositions, pageWork } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageWork.headerImage}
              messageBgColor={pageWork.headerTextBgColor}
              messageOne={pageWork.headerText1}
              messageTwo={pageWork.headerText2}
            />

            <GridRow>
              <FilterByLocation>
                {location => (
                  <React.Fragment>
                    {openpositions
                      .filter(filterByLocation(location))
                      .map(annons => (
                        <Block
                          data-test="location"
                          key={annons.title}
                          id={annons.id}
                          title={annons.title}
                          subtitle={annons.location}
                        >
                          {annons.role}
                        </Block>
                      ))}
                  </React.Fragment>
                )}
              </FilterByLocation>
            </GridRow>

            <Team bgColor="red" shortName={['rln', 'msr']} />
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Work
