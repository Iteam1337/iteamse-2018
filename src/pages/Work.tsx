import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { WorkPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import FilterByLocation from '../components/FilterByLocation/FilterByLocation'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import StyledPrefetchLink from '../components/Link/StyledPrefetchLink'
import Team from '../components/Team/Team'
import { filterByLocation } from '../utils/filterByLocation'
import { set } from '../utils/googleAnalytics'
import { OPEN_POSITION_PAGE_QUERY } from './OpenPosition'

export const WORK_PAGE_QUERY = gql`
  query WorkPage {
    pageWork {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      contactTitle
      team
    }
    openpositions {
      id
      location
      role
      title
    }
  }
`

class WorkQuery extends Query<WorkPageQuery> {}

export class Work extends React.Component {
  componentDidMount() {
    set('jobba-hos-oss')
  }
  render() {
    return (
      <>
        <Helmet>
          <title>Iteam - There's a better way | Jobba hos oss</title>
          <meta
            name="og:title"
            content="Iteam - There's a better way | Jobba hos oss"
          />
          <meta
            name="twitter:title"
            content="Iteam - There's a better way | Jobba hos oss"
          />
        </Helmet>
        <WorkQuery query={WORK_PAGE_QUERY}>
          {({ loading, data }) => {
            if (loading || !data) {
              return null
            }

            const { openpositions, pageWork } = data

            return (
              <>
                <Header
                  backgroundImage={pageWork.headerImage}
                  messageBgColor={pageWork.headerTextBgColor}
                  messageOne={pageWork.headerText1}
                  messageTwo={pageWork.headerText2}
                />

                <GridColumn>
                  <FilterByLocation>
                    {location => (
                      <>
                        {openpositions
                          .filter(filterByLocation(location))
                          .map(annons => (
                            <Block
                              data-test="location"
                              key={annons.title}
                              readMore={
                                <StyledPrefetchLink
                                  query={OPEN_POSITION_PAGE_QUERY}
                                  to={`/jobba-hos-oss/${annons.id}`}
                                  variables={{
                                    id: annons.id,
                                  }}
                                >
                                  Läs mer
                                </StyledPrefetchLink>
                              }
                              subtitle={annons.location}
                              title={annons.title}
                            >
                              {annons.role}
                            </Block>
                          ))}
                      </>
                    )}
                  </FilterByLocation>
                </GridColumn>

                <Team
                  bgColor="red"
                  callToAction={pageWork.contactTitle}
                  shortName={pageWork.team}
                />
              </>
            )
          }}
        </WorkQuery>
      </>
    )
  }
}

export default Work
