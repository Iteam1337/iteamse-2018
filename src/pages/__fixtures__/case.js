export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
