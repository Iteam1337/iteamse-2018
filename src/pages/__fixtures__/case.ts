export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
