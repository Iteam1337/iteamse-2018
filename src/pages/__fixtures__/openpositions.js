export const openpositions = [
  {
    id: 'junior-ux-designer',
    location: 'Stockholm',
    role:
      'Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.\n\nDu delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. ',
    title: 'Junior UX Designer',
    __typename: 'OpenPosition',
  },
]
