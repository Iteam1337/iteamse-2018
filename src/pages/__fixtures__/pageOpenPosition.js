export const pageOpenPosition = {
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2VBOGPr664aWq4cEieYasY/7d824e86c44854559806e31e02c2e02a/juniorux.jpg',
  headerText1: 'UX Designer',
  headerText2: 'Stockholm',
  headerTextBgColor: '#668CFF',
  application:
    'Maila CV, personligt brev och beskrivning av ett case till: [work@iteam.se](mailto:work@iteam.se)',
  applicationTitle: 'Ansökan',
  aboutUs:
    'Vi digitaliserar företag och organisationer genom strategi, kod och kultur. Med hjälp av de tre bitarna skapar vi innovativa tekniska lösningar som skapar värde, gör något bra för samhället och låter oss ha roligt längs vägen.\n\nVåra kunder har spännande utmaningar både tekniskt och affärsmässigt, och stor potential att göra skillnad inom sina områden. Vi jobbar bland annat med Taxi Stockholm, Vimla, Läkare utan gränser och Arbetsförmedlingen.\n\nVi är ca 30 personer fördelade i Stockholm och Göteborg. Hos oss tar du av dig skorna vid dörren och känner dig som hemma. Vi lär oss ständigt nya saker av varandra - genom mentorskap, utbyte inom projekt, måndagsfrukostar, kodlunch eller under labbtimmarna. Vi har kul tillsammans och tar hand om varandra i vardagen, löparspåret, på yogamattan, spelkvällar, after works och vid andra speciella tillfällen. Vi arbetar aktivt för att sätta ihop team med så varierande och kompletterande bakgrund som möjligt – oavsett om det gäller kön, könsidentitet och -uttryck, etnisk tillhörighet, religion och trosuppfattning, funktionalitet, sexuell läggning, ålder, utbildning eller social bakgrund. Våra team är alltid korsfunktionella. Alltid agila. Alltid nära beställaren, deras anställda, användare och intressenter.',
  aboutUsTitle: 'Om oss',
  bonusKnowledge:
    '- Kvantitativa researchmetoder (t.ex. enkäter och användningsdata i Google Analytics)\n- Förståelse för den tekniska miljö dina lösningar kommer att implementeras i (t.ex. grundläggande HTML och CSS)\n- Utbildning inom interaktionsdesign, människa-datorinteraktion, beteendevetenskap, kognitionspsykologi, digital design eller närliggande område',
  bonusKnowledgeTitle: 'Plus i kanten',
  contentImage:
    '//images.ctfassets.net/rj4r6yfcesw5/62hgzE8G3ueMa2YSEuG4og/db2b643db6041f9d26936bc2d6006607/bleed_howwework.jpg',
  knowledge:
    '- Kvalitativa researchmetoder (t.ex. användbarhetstester, observationer, intervjuer och workshops)\n- Kommunicera dina insikter med hjälp av lämpliga leverabler (t.ex. rapporter, användarresor, prototyper eller skrynkliga pappersskisser)\n- Interaktionsdesign\n- Vara användarens representant i projekt och stötta både kollegor och kunder med användarens perspektiv\n- Jobba agilt\n- Arbeta korsfunktionellt och förstå behoven hos kollegor ur andra yrkesgrupper\n- Goda kunskaper i engelska (för att kunna kommincera med dina kollegor)\n- Goda kunskaper i svenska (för att kunna genomföra användarstudier i kontexter där det primära användarspråket är svenska)',
  knowledgeTitle: 'Du behöver kunna',
  location: 'Stockholm',
  role:
    'Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.\n\nDu delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. ',
  roleTitle: 'Din roll',
  technicalities:
    '__Några av våra kunder:__ Taxi Stockholm, Vimla, Swedbank, Läkare Utan Gränser, Arbetsförmedlningen, Alcro Beckers, Scania, SoundIdeas.\n\n__Anställningsform:__ Tillsvidareanställning. 6 månaders provanställning, sedan fast anställning.\n\n__Plats:__ Stockholm.\n\n__Start:__ Omgående.',
  technicalitiesTitle: 'Teknikaliteter',
  title: 'Junior UX Designer',
  __typename: 'OpenPosition',
}
