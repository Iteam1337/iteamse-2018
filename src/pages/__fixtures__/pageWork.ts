export const pageWork = {
  __typename: 'PageWork',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/j9IXvqaJ32Uc0SWQeUQsi/e75bc2d9b05ae39fe71ec1b1c14600e0/work.jpg',
  headerText1: 'Bli en del av teamet',
  headerText2: null,
  headerTextBgColor: '#FF405F',
}
