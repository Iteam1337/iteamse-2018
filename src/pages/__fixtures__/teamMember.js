export const teamMember = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'New York',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    __typename: 'TeamMember',
  },
]
