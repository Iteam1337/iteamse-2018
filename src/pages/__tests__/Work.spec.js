import React from 'react'
import { Work } from '../Work'
import { render, Simulate, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageWork } from '../__fixtures__/pageWork'
import { teamMember } from '../__fixtures__/teamMember'
import { openpositions } from '../__fixtures__/openpositions'
import { pageOpenPosition } from '../__fixtures__/pageOpenPosition'

describe('components/Work', () => {
  const mockedResponse = {
    openpositions,
    pageOpenPosition,
    pageWork,
    teamMember,
  }

  it('renders Work', async () => {
    const { container } = render(
      <MockedQuery response={mockedResponse}>
        <Work />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })

  it('should handle prefetching any work when hovering link', async () => {
    const client = {
      query: jest.fn(),
    }

    const { getByText } = render(
      <MockedQuery response={mockedResponse}>
        <Work client={client} />
      </MockedQuery>
    )

    await wait()

    Simulate.mouseEnter(getByText('LÃ¤s mer'))

    expect(client.query.mock.calls[0][0]).toMatchSnapshot()
  })
})
