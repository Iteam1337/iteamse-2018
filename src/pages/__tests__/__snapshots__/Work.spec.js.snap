// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Work renders Work 1`] = `
<div>
  <div
    style="height: 700px;"
    class="lazyload-placeholder"
  />
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD iyGAEe"
  >
    <div
      class="s3nsc5b-0-PaddedRow-hWTqUj iUATLn"
    >
      <div
        class="Filter__Wrap-s16ccbkk-0 evelsV"
        data-test="filter-"
        tabindex="0"
      >
        Alla
      </div>
      <div
        class="Filter__Wrap-s16ccbkk-0 lgKsfF"
        data-test="filter-Stockholm"
        tabindex="0"
      >
        Stockholm
      </div>
      <div
        class="Filter__Wrap-s16ccbkk-0 lgKsfF"
        data-test="filter-Göteborg"
        tabindex="0"
      >
        Göteborg
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 kZVCTc"
        data-test="block-location"
      >
        <div
          class="Block__Content-jdahsb-0 jnCWxO"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bXouBG"
            >
              Junior UX Designer
            </h3>
            <div
              class="Block__Subtitle-jdahsb-2 enSAoq"
            >
              Stockholm
            </div>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.
                </p>
                <p>
                  Du delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. 
                </p>
              </div>
            </div>
            <a
              class="Link__StyledLink-kqcp4a-0 exsHFb"
              href="/jobba-hos-oss/junior-ux-designer"
            >
              Läs mer
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD hcBJZm"
  >
    <div
      class="TeamMembers__Members-gavpkf-0 uMipK"
    >
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 XxNhO"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cmm"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              Cookie master
            </div>
            <div>
              Stockholm
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Cookie M
          </div>
          <div>
            <a
              class="s1qo6rwq-0-NativeLink-jXoiCT dJdJnO"
              href="tel:0701234567"
            >
              070-123 45 67
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:cm@sesame.com"
          >
            cm@sesame.com
          </a>
        </div>
      </div>
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 XxNhO"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cvc"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              1 ahahah
            </div>
            <div>
              Göteborg
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Count van Count
          </div>
          <div>
            <a
              class="s1qo6rwq-0-NativeLink-jXoiCT dJdJnO"
              href="tel:0707654321"
            >
              070-765 43 21
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:count@sesame.com"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
